### ISSUE TITLE
Telemetry via Jaeger results in DSC start abort

### ISSUE THREAD

**Issue by ghost** [International-Data-Spaces-Association/DataspaceConnector#316]
Switching on the telemetry collection via Jaeger leads to a start abort of the DSC.
`java.lang.NoSuchMethodError: void org.apache.thrift.transport.AutoExpandingBufferWriteTransport`
Docu: https://international-data-spaces-association.github.io/DataspaceConnector/Deployment/Configuration/Telemetry

Reason: The (deprecated) libthrift dependency required by the current used tracing-implementation [is excluded](https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/5872b8eb21d4883c2e9319e8d77a028e3bfcc73b/pom.xml#L410) from opentracing via the POM.
```
<exclusion>
  <artifactId>libthrift</artifactId>
  <groupId>org.apache.thrift</groupId>
</exclusion>
```

Instead, a newer version of the libthrift dependency is included transitively via the IDS-serializer.
```
+- de.fraunhofer.iais.eis.ids:infomodel-serializer:jar:4.2.8:compile
  +- org.apache.jena:jena-arq:jar:4.1.0:compile
    +- org.apache.thrift:libthrift:jar:0.14.1:compile
```

Unfortunately the implemented opentracing solution used is not compatible with this newer libthrift version.

Workaround for testing telemetry collection via Jaeger (do not use productively due to deprecated libthrift version): Exclude the libthrift dependency of the IDS-serializer and use the opentracing version instead.

The bottom line is that another solution should be used for telemetry collection.

**Comment by ghost** [comment]
Will replace the old OpenTracing with the newer OpenTelemetry with custom traces & spans.

**Comment by ghost** [comment]
Preview (WIP):
E.g. sending of DescriptionRequestMessages and waiting for a response:
From the second call on, the cached DAT is used (and saves the DAPS call) and the SHACL shapes do not need to be downloaded again, which reduces the complete duration from sending the request to processing the response from 2 seconds to 0,14 seconds.

![image](https://user-images.githubusercontent.com/75306992/175280051-7faae654-4f83-414a-b3e5-fca143e3317e.png)

Update (24.06.22):
![image](https://user-images.githubusercontent.com/75306992/175557569-eb27fc03-e62b-4740-8125-c2436d7d4f26.png)
